modules/serverless-create-eventing-namespace.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+//  * serverless/installing_serverless/installing-knative-eventing.adoc
+
+[id="serverless-create-eventing-namespace_context"]
+= Creating the `knative-eventing` namespace
+
+To create the `knative-eventing` namespace, you must create a `knative-eventing` project.
```

Comment:
This isn't a big problem, but someone could nitpick on that as I do. This sentence isn't 100% true:
> To create the `knative-eventing` namespace, you must create a `knative-eventing` project.

To create a namespace we can also create a namespace, not nesseserly a project. And in OCP projects and namespaces are basically the same thing.

Also, both namespaces  and projects can be easily created from administrative console and command line. From console, it's: Administration > Namespaces > Create namespace - project will also be created. For console, it's `oc create namespace knative-eventing`.